backend/app/notify/email.go

```diff
@@ -0,0 +1,137 @@
+package notify
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"net/smtp"
+	"time"
+
+	log "github.com/go-pkgz/lgr"
+	"github.com/pkg/errors"
+)
+
+// EmailParams contain settings for email set up
+type EmailParams struct {
+	Host     string        // SMTP host
+	Port     int           // SMTP port
+	TLS      bool          // TLS auth
+	From     string        // From email field
+	Username string        // user name
+	Password string        // password
+	TimeOut  time.Duration // TLS connection timeout
+}
+
+// Email implements notify.Destination for email
+type Email struct {
+	connection *smtp.Client
+	host       string        // SMTP host
+	port       int           // SMTP port
+	tls        bool          // TLS auth
+	from       string        // From email field
+	username   string        // user name
+	password   string        // password
+	timeOut    time.Duration // TLS connection timeout
+}
+
+const emailConnectionTimeOut = 10 * time.Second
+
+//NewEmail makes email object for notifications and run sending daemon
+func NewEmail(params EmailParams) (*Email, error) {
+
+	res := Email{
+		host:     params.Host,
+		port:     params.Port,
+		tls:      params.TLS,
+		from:     params.From,
+		username: params.Username,
+		password: params.Password,
+		timeOut:  params.TimeOut,
+	}
+	if res.timeOut == 0 {
+		res.timeOut = emailConnectionTimeOut
+	}
+	log.Printf("[DEBUG] create new email notifier for server %s with user %s, timeout=%s",
+		res.host, res.username, res.timeOut)
+
+	var err error
+	res.connection, err = res.client()
+
+	return &res, err
+}
+
+// Send email from request to address in settings
+func (e *Email) Send(ctx context.Context, req request) error {
+	log.Printf("[DEBUG] send notification via %s, comment id %s", e, req.comment.ID)
+	msg := e.buildMessageFromRequest(req, "test@localhost")
+	return e.SendEmail(msg)
```

Comment:
I think you missed auth. Not sure if you planned to do it in Send or other place, but will need smth like this anyway https://github.com/go-pkgz/auth/blob/master/provider/sender/email.go#L78